scripts/install_texlive.sh

```diff
@@ -45,6 +43,8 @@ cd ..
 rm -rf install-tl-*
 rm /tmp/texlive-profile.txt
 
+ln -rs /usr/local/texlive/bin/${ARCH}-linux /usr/local/texlive/bin/linux
```

Comment:
Is there a reason you are specifying the `-r` option?

Reply:
In such a case I prefer a relative link.

Reply:
Note that other scripts does not use the `-r` option now.

https://github.com/rocker-org/rocker-versioned2/blob/ae741564c8c965672d329a94e7930c4e4ba98809/scripts/install_pandoc.sh#L53

Reply:
> In such a case I prefer a relative link.

such a case = a symbolic link within the same directory.

Reply:
I see. I have no preference myself.
Certainly the `-r` option might be reasonable for the same directory.

Given the possibility of repeated execution, add the `-f` option seems a good idea.